web_external/js/views/body/StudyView.js

```diff
@@ -0,0 +1,26 @@
+isic.views.StudyView = isic.View.extend({
+    initialize: function (settings) {
+        girder.cancelRestRequests('fetch');
+
+        if (settings.study) {
+            this.study = settings.study;
+            this.render();
+        } else if (settings.id) {
```

Comment:
Generally, I think a View should be responsible for fetching its own Model with details. However, I've left it an option to pass in a pre-fetched Model.

@msmolens What do you think of this policy in general?